src/gatsby/src/components/park/about.js

```diff
@@ -43,16 +43,16 @@ export default function About({
                   <strong>Size:</strong>
                   {marineProtectedArea === "Y" ?
                     <>
-                      {" "}{park.totalArea} hectares
+                      {" "}{park.totalArea.toLocaleString()} hectares
```

Comment:
I suggest we force the locale to en-CA
`park.totalArea.toLocaleString("en-CA")`
Otherwise, we are at the mercy of the user's browser settings.